TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -50,42 +49,73 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
         }
 
         view.findViewById<FloatingActionButton>(R.id.add_fab).setOnClickListener {
```

Comment:
should we create an explicit variable for the value like `val addButton = view.findViewById`? Not be necessary, but might save time if we want to add features to it later

Reply:
I think it may not be necessary for this control specifically (because the floating action button should only have one purpose) but I will definitely keep this in mind for future controls!